docs/process.md

```diff
@@ -173,12 +173,12 @@ Their full descriptions can be viewed under the [labels page](https://github.com
 #### Code review
 
 Your code will be reviewed, in this sequence, by:
-* Travis CI: by running static analysis.<br>
+* Github Actions: by running static analysis.<br>
   If there are problems found, the build will terminate without proceeding to testing.<br>
```

Comment:
I don't think this line is accurate anymore; it was in V6

Reply:
yup removed